docs/migration/migrate_6_0.asciidoc

```diff
@@ -29,3 +29,9 @@ This is no longer the case. Now, only commas are a valid query separator: e.g. `
 
 
 *Impact:* You will need to migrate your stored timelion queries to the new syntax.
+
+[float]
+=== Removed same port http to https redirect behavior
+*Details:* Kibana 5.x redirected requests from http to https on the same port if TLS was configured.  This was removed.
+
+*Impact:* You may need to add a redirect on a different port if a redirect is expected.
```

Comment:
I know the plan here is to follow up with a different PR to handle port redirection stuff, but when this specific PR is merged, the reality is going to be that there is no way to do http -> https redirection in Kibana.  We should update the breaking changes impact to mention that the only way to accomplish this would be to custom handle http -> https redirection in a reverse proxy or at the load balancer.